core/src/main/scala/org/broadinstitute/dsde/rawls/model/ErrorReport.scala

```diff
@@ -0,0 +1,5 @@
+package org.broadinstitute.dsde.rawls.model
+
+object ErrorReport extends ErrorReportBase {
+  def SOURCE = "rawls"
+}
```

Comment:
+1 to doug's comment. This is the service-specific class, so it can/should have a service-specific name.